landing/tests/tests.py

```diff
@@ -0,0 +1,15 @@
+from django.test import Client
+
+ENDPOINT = 'learn-more'
+
+
+def test_client():
+    client = Client()
+    response = client.get(f'/{ENDPOINT}/')
+    assert response.status_code == 200
+    response = client.get(f'/{ENDPOINT}')
```

Comment:
remove all tests from line 11
Maybe just check that the file itself has returned

Reply:
> remove all tests from line 11 Maybe just check that the file itself has returned

Resolved

Reply:
Please check the correct template returned, review previous talk on ui testing

Reply:
> Please check the correct template returned, review previous talk on ui testing

how can i check this?

Reply:
> Please check the correct template returned, review previous talk on ui testing

[Used as shown here](https://stackoverflow.com/a/62551690)

I used functions for maintainability